codexctl/updates.py

```
@@ -96,24 +97,24 @@ def update_version_ids(self, location: str) -> None:
-                raise SystemExit(
-                    "Connection timed out while downloading version-ids.json! Do you have an internet connection?"
-                )
-            except Exception as error:
-                raise SystemExit(
-                    f"Unknown error while downloading version-ids.json! {error}"
-                )
+                _ = f.write("\n")
+        except requests.exceptions.Timeout:
+            raise SystemExit(
+                "Connection timed out while downloading version-ids.json! Do you have an internet connection?"
+            )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)